inventory_management_api/config.py

```diff
@@ -0,0 +1,8 @@
+def configure_app(app):
+    app.config["REDIS_HOST"] = "localhost"
+    app.config["REDIS_PORT"] = 6379
+    app.config["DB_HOST"] = "localhost"
+    app.config["DB_USER"] = "user"
+    app.config["DB_PASSWORD"] = "password"
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/trulyfurqan/eshop-fastapi/pull/7#issuecomment-2226005304>#907ae7</a><span> - 07/12/2024, 05:29 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Possible hardcoded password assigned to: "DB_PASSWORD"
</div>
<div id="fix">
<b>Fix:</b> Avoid hardcoding sensitive information like passwords. Use environment variables or a secure vault.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,6 +1,6 @@
      app.config["REDIS_PORT"] = 6379
      app.config["DB_HOST"] = "localhost"
      app.config["DB_USER"] = "user"
 -    app.config["DB_PASSWORD"] = "password"
 +    app.config["DB_PASSWORD"] = os.getenv("DB_PASSWORD", "default_password")
      app.config["DB_NAME_1"] = "inventory_db1"
      app.config["DB_NAME_2"] = "inventory_db2"
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged